chroot-bin/croutoncycle

```diff
@@ -123,7 +123,7 @@ if [ "$tty" = 'tty1' ]; then
     for disp in $winlist; do
         if [ "${disp%"*"}" != "$disp" ]; then
             curdisp="$disp"
-            if [ -n "$xiwiactive" -a "${disp%"*"}" = "$aurawin" -a \
+            if [ -n "$xiwiactive" -a "$aurawin" != "${aurawin%${curdisp%"*"}*}" -a \
```

Comment:
Actually, i'm worried about substring matches here.  How about making the aurawin list with a guaranteed space on either side of each entry, and then including the guard spaces in the suffix remove operations?